File: modules/sample-springMvc/src/test/java/examples/server/springMvc/pet/MockPetHandlerConfiguration.java
package examples.server.springMvc.pet;

import org.mockito.Mockito;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
public class MockPetHandlerConfiguration {
    @Bean
    public PetHandler nameService() {
        return Mockito.mock(PetHandler.class);
    }
}

File: modules/sample-springMvc/src/test/scala/examples/server/springMvc/pet/MockPetHandlerConfiguration.scala
package examples.server.springMvc.pet

import org.mockito.Mockito
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration class MockPetHandlerConfiguration {
  @Bean def nameService: PetHandler = Mockito.mock(classOf[PetHandler])
}
